### PR TITLE
feat: TRADE-3243 update circle-question-mark icons

### PR DESCRIPTION
**Pull request by ascagnel**

### CODE REVIEW

File: src/circle-question-mark-empty.svg [hunk 1]
```diff
@@ -1,5 +1,11 @@
...
+<svg xmlns="http://www.w3.org/2000/svg" class="dibsvg-svg dibsvg-circle-question-mark-empty" viewBox="0 0 250 250">
+<g>
+	<path d="M125-0.2C56.1-0.2,0,55.9,0,124.8s56.1,125,125,125s125-56,125-125C250,55.9,193.9-0.2,125-0.2z M125,240.9
```
Comment (hellatan) [inline src/circle-question-mark-empty.svg]: the linter didn't complain about this? I feel like you don't need the wrapping `<g>` tag and all of these properties should be on on line.

File: src/circle-question-mark-empty.svg [hunk 2]
```diff
@@ -1,5 +1,11 @@
...
+		c-64,0-116-52.1-116-116c0-64,52.1-116,116-116s116,52,116,116S189,240.9,125,240.9z"/>
+	<path d="M124.2,69.4c-10.8,0-19.5,3.4-26.1,10.1c-6.6,6.7-10.2,14.9-10.7,24.5l18.1,2.6c1.3-6.7,3.6-11.7,7-15
+		c3.4-3.3,7.6-4.9,12.6-4.9c5.2,0,9.3,1.6,12.4,4.7c3.1,3.1,4.6,6.9,4.6,11.3c0,3.2-0.9,6.1-2.6,8.7c-1.1,1.7-4.5,5.2-10.3,10.6
+		c-5.7,5.4-9.6,10.2-11.5,14.5s-2.9,9.8-2.9,16.5c0,0.6,0.1,2.4,0.1,5.4h17.9c-0.1-6.2,0.3-10.6,1.3-13c1-2.4,3.6-5.6,7.8-9.6
+		c8.1-7.6,13.3-13.7,15.8-18.2s3.7-9.2,3.7-14.2c0-9.1-3.4-17-10.1-23.8C144.6,72.8,135.5,69.4,124.2,69.4z"/>
+	<ellipse transform="matrix(0.9732 -0.2298 0.2298 0.9732 -37.8237 33.3495)" cx="124.3" cy="179.1" rx="11.3" ry="11.3"/>
```
Comment (hellatan) [inline src/circle-question-mark-empty.svg]: what does this make? The outside circle or the dot to the question mark?

File: src/circle-question-mark-filled.svg [hunk 3]
```diff
@@ -1,3 +1,9 @@
...
+<svg xmlns="http://www.w3.org/2000/svg" class="dibsvg-svg dibsvg-circle-question-mark-filled" viewBox="0 0 250 250">
+<path d="M125,249.8c-68.9,0-125-56.1-125-125c0-68.9,56.1-125,125-125c68.9,0,125,56.1,125,125C250,193.8,193.9,249.8,125,249.8z
```
Comment (hellatan) [inline src/circle-question-mark-filled.svg]: try running these through the svgo linter. This formatting is strange.

File: src/circle-question-mark-empty.svg (hunk 2 above)
Comment (ascagnel) [reply src/circle-question-mark-empty.svg]: this is the question mark dot

**Comment by ascagnel** [comment]
bump -- ran the two new SVGs through `svgo`

**Comment by ascagnel** [comment]
i'll hold off until tomorrow morning to merge this -- looks like GH still has some issues:

```
16:03 EDT
We continue working through the backlogs for Pages builds and webhook deliveries, and we are monitoring as the site recovers.
```

**Comment by hellatan** [comment]
this repo isn't setup with important webhooks so this is safe to merge.

**Comment by hellatan** [comment]
pubished `dibs-vg@1.80.0`

**Pull request merged**